lib/iris/tests/results/analysis/aggregated_by/multi_missing.cml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
-  <cube core-dtype="float64" dtype="float64" fill_value="999999.0" long_name="temperature" units="kelvin">
```

Comment:
Is removing `fill_value` from the CML desirable? It's obvious why it's happened, but is it what we want? The CML now stores less metadata about the cube so the CML may now be less valuable as well.